base.css

```diff
@@ -94,6 +103,24 @@ button {
   align-items: flex-start;
 }
```

Comment:
1.Extra lines can be removed
2. `align-items` is repeated.

Reply:
I want to have 3-4 categories of align-items like flex-start,centre etc.,
That's why repeated

Reply:
I want to have 3-4 categories of align-items like flex-start,centre etc.,
That's why repeated